infra/build_specified_commit.py

```diff
@@ -22,88 +22,98 @@
 import subprocess
 
 import helper
-import repo_manager
 
 
-class DockerExecutionError(Exception):
-  """An error that occurs when running a docker command."""
+class BuildData():
+  """List of data requried for bisection of errors in OSS-Fuzz projects.
```

Comment:
Maybe drop "List" and just start with "Data".
Also "required".

Reply:
Done.